src/domain/organizations/organizations-safe.repository.spec.ts

```diff
@@ -0,0 +1,679 @@
+        status: faker.helpers.arrayElement(
+          getStringEnumKeys(OrganizationStatus),
+        ),
+        name: faker.word.noun(),
+      });
+      const orgId = org.identifiers[0].id as Organization['id'];
+
+      await expect(
+        orgSafesRepo.findOrFail({
+          where: { organization: { id: orgId } },
+        }),
+      ).rejects.toThrow('Organization has no Safes.');
+    });
+  });
+
+  describe('find', () => {
+    it('should return found organization Safes', async () => {
+      const orgSafes = faker.helpers.multiple(
+        () => ({
+          chainId: faker.string.numeric(),
+          address: getAddress(faker.finance.ethereumAddress()),
+        }),
+        { count: { min: 2, max: 5 } },
+      );
+      const org = await dbOrgRepo.insert({
+        status: faker.helpers.arrayElement(
+          getStringEnumKeys(OrganizationStatus),
+        ),
+        name: faker.word.noun(),
+      });
+      const orgId = org.identifiers[0].id as Organization['id'];
+      await orgSafesRepo.create({
+        organizationId: orgId,
+        payload: orgSafes,
+      });
+
+      await expect(
+        orgSafesRepo.find({
+          where: { organization: { id: orgId } },
+        }),
+      ).resolves.toEqual(
+        expect.arrayContaining(
+          orgSafes.map(({ chainId, address }) => ({
+            id: expect.any(Number),
+            chainId,
+            address,
+            createdAt: expect.any(Date),
+            updatedAt: expect.any(Date),
+          })),
+        ),
+      );
+    });
+
+    it('should return empty array if no organization Safes found', async () => {
+      const org = await dbOrgRepo.insert({
+        status: faker.helpers.arrayElement(
+          getStringEnumKeys(OrganizationStatus),
+        ),
+        name: faker.word.noun(),
+      });
+      const orgId = org.identifiers[0].id as Organization['id'];
+
+      expect(
+        await orgSafesRepo.find({
+          where: { organization: { id: orgId } },
+        }),
+      ).toEqual([]);
+    });
+  });
+
+  describe('delete', () => {
+    it('should delete an OrganizationSafe', async () => {
+      const org = await dbOrgRepo.insert({
+        status: faker.helpers.arrayElement(
+          getStringEnumKeys(OrganizationStatus),
+        ),
+        name: faker.helpers.arrayElement([
+          faker.word.noun(),
+          faker.word.noun(),
+        ]),
+      });
+      const orgId = org.identifiers[0].id as Organization['id'];
+      const orgSafe = await dbOrgSafeRepo.insert({
+        chainId: faker.string.numeric(),
+        address: getAddress(faker.finance.ethereumAddress()),
+        organization: {
+          id: orgId,
+        },
+      });
+      const orgSafeId = orgSafe.identifiers[0].id as OrganizationSafe['id'];
+
+      const orgSafeBefore = await orgSafesRepo.findOrFail({
+        where: { id: orgSafeId },
+      });
+      expect(orgSafeBefore).toEqual(
+        expect.arrayContaining([
```

Comment:
We don't need to use `expect.arrayContaining` here.

Reply:
Removed in [219808a](https://github.com/safe-global/safe-client-gateway/pull/2365/commits/219808a85cb3e08a769de9ac1b54a8b6c8b44a86)